bude_hezky/weather/weather_resolver.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python3
```

Comment:
tohle není potřeba pokud nechceš pouštět ten soubor jako skript v bashi přes `chmod +x` (a ani potom to není úplně dobrý dělat, ale to je na celou přednášku)

Reply:
Jj, já už to odstraňoval, ale zapomněl jsem všude.